botfront/imports/ui/components/nlu/common/style.import.less

```diff
@@ -83,7 +83,19 @@
     }
 }
 
-.ui.popup.intent-popup {
+.ui.modal.intent-popup {
+    &:before {
+        position: absolute;
+        top: -100px;
```

Comment:
perhaps -30px would work better? -100px feels a bit far to me
<img width="324" alt="Screen Shot 2020-05-15 at 12 27 28 PM" src="https://user-images.githubusercontent.com/49006070/82076109-52be5200-96ab-11ea-8cf0-7c115444fd2a.png">